app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java

```diff
@@ -36,6 +52,13 @@ public AppiumResponse safeHandle(IHttpRequest request) {
                     || "resourceId".equals(attributeName)) {
                 String attribute = element.getStringAttribute(attributeName);
                 return new AppiumResponse(getSessionId(request), WDStatus.SUCCESS, attribute);
+            } else if ("contentSize".equals(attributeName)) {
+                Rect boundsRect = element.getBounds();
+                ContentSize contentSize = new ContentSize(boundsRect);
```

Comment:
does it make sense to check if boundsRect isEmpty here?

Reply:
i don't see why; we would still want to return the 0-dimension info as the response